### PR TITLE
Edited GUI Frames

### PR DESCRIPTION
**Pull request by Poockins**
Made corrections to the GUI frames, added in authors, sizing and formatting. Welcome Screen needs to be centered.

### CODE REVIEW

File: src/main/java/calculator/PokerCalculator.java [hunk 1]
```diff
@@ -1,3 +1,6 @@
+package calculator;
+
+import calculator.Cards;
```
Comment (vlyandra) [inline src/main/java/calculator/PokerCalculator.java:3]: This should be unnecessary because they're in the same package.

**Review by vlyandra — CHANGES_REQUESTED** [review]
LGTM other than one thing, see review.

File: src/main/java/calculator/PokerCalculator.java (hunk 1 above)
Comment (Poockins) [reply src/main/java/calculator/PokerCalculator.java:3]: For some reason, I was not able to get the code to run except when I implemented this import. I also had to move the "package calculator;" above all the imports for it to run. Not really sure why

Comment (vlyandra) [reply src/main/java/calculator/PokerCalculator.java:3]: Package declarations always need to be the first source line in Java.

**Review by vlyandra — APPROVED** [review]

**Pull request merged**